components/raftstore/src/store/fsm/apply.rs

```diff
@@ -2292,6 +2296,27 @@ impl Apply {
             commit_term,
         }
     }
+
+    fn get_total_mem_size(&self) -> i64 {
```

Comment:
Better cache it instead of calculate it multiple times.

Reply:
I think maybe it's ok. Because we only calculate it when the apply droping.

Reply:
+1, we could cache it inside the struct.

Reply:
Cached.